src/interfaces/task.ts

```diff
@@ -19,6 +19,7 @@ export interface CreateExportTaskRequest<T> {
 
 export interface CreateExportTaskResponse {
   jobId: string;
+  ROI: FeatureCollection;
```

Comment:
for domains that doesn't need ROI shouldn't it be nullable?

Reply:
true, fixed changed to nullable